src/desktop_pixmap.hpp

```diff
@@ -34,11 +34,8 @@ class desktop_pixmap : public drawable
   );
   ~desktop_pixmap ();
 
-  // Explicitly deleting unused constructors to comply with rule of five
-  desktop_pixmap (const desktop_pixmap &other) = delete;
-  desktop_pixmap (desktop_pixmap &&other) noexcept = delete;
-  desktop_pixmap &operator= (const desktop_pixmap &other) = delete;
-  desktop_pixmap &operator= (desktop_pixmap &&other) = delete;
+  // Declaring copy operator to use it in vector
+  desktop_pixmap (const desktop_pixmap &);
```

Comment:
Are you sure this one covers all the rule of five?

Reply:
iirc when you explicitly declare copy constructor, move constructor won't be declared implicitly
same for assignment operator